pages/contact.html

```diff
@@ -98,8 +98,11 @@ <h4><a href="#ml">Mailing lists</a></h4>
                 <li><a href="https://lists.opensuse.org/archives/list/translation@lists.uyuni-project.org/" target="_blank">translation</a>: To discuss about translations (anyone can post as long as it is registered).</li>
             </ul>
             <h4><a href="#bugs">Reporting bugs</a></h4>
-            <p> Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
-            <div class="alert alert-warning" role="alert"> Verify that a duplicate of the issue does not already exist! </div>
+            <p>Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
+            <p>Please verify that a duplicate of the issue does not already exist!</p>
```

Comment:
Why are removing the warning? I'd leave it.

Reply:
It's so prominent it effectively hides everything around it. 

In fact, I want to remove the warnings from https://www.uyuni-project.org/pages/stable-version.html too: when you access that page, it's very hard to notice the release notes because your eyes go immediately go the yellow boxes.

Reply:
OK, but changing it to a simple `<p>` makes it no prominent at all, and this is not even part of this PR (which is about adding the community hours). Either rollback this and we can think later about a proper solution, or propose an alternative that still highlights that this is important :-)

Reply:
OK, but changing it to a simple `<p>` makes it no prominent at all, and this is not even part of this PR (which is about adding the community hours). Either rollback this and we can think later about a proper solution, or propose an alternative that still highlights that this is important :-)

Reply:
Making it not prominent at all is on purpose. It's not important enough to hide the rest of the page.

I'm going to move it to another PR, where I would remove the yellow box in stable-version.html and this page, just for the sake of both highlight-removals to be together.

Reply:
Making less prominent is OK, making it not prominent is not OK.